ecommerce/extensions/payment/views/payu.py

```diff
@@ -89,9 +82,13 @@ def get(self, request):
         logger.info(u"PayU payment [%s] transactionState [%s]", payment_id, transactionState)
 
         payu_response = request.GET.dict()
-        basket = self._get_basket(payment_id)
-
-        if not basket:
+        try:
```

Comment:
This is inspired on our Payu implementation right?

Reply:
Yes.